Services/Form/templates/default/tpl.prop_datetime_duration.html

```diff
@@ -26,7 +27,8 @@
 		<!-- BEGIN end_label_bl -->
 		<!-- END end_label_bl -->
 		<div class="input-group date<!-- BEGIN end_width_bl --> noop<!-- END end_width_bl -->" id="{DATEPICKER_END_ID}">
-			<input aria-label="{END_ARIA_LABEL}" type="text" class="form-control" style="min-width:120px" placeholder="{END_PLACEHOLDER}" value="{DATEPICKER_END_VALUE}" name="{DATE_END_ID}"{DATEPICKER_END_DISABLED} {END_REQUIRED} />
+			<input aria-label="{END_ARIA_LABEL}" aria-describedby="end_date_desc" type="text" class="form-control" style="min-width:120px" placeholder="{END_PLACEHOLDER}" value="{DATEPICKER_END_VALUE}" name="{DATE_END_ID}"{DATEPICKER_END_DISABLED} {END_REQUIRED} />
+			<p id="end_date_desc" style="display: none;">{DESCRIPTION}</p>
```

Comment:
I hope the inline css is okay here. It will prevent possible flickering because it has effect right away. By the way, inline css is already use two lines above in this template.